wordpress-mobile/WordPressAuthenticator-iOS
# Update CocoaPods to the 1.7 series for Xcode 11

It's all in the title 😃 

**To Test:**
- Run `rm -rf Pods && bundle install && bundle exec pod install && bundle exec pod lib lint` on this branch to ensure that pods can be properly installed and built.
